apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -0,0 +1,299 @@
+const pino = require('pino')
+const dayjs = require('dayjs')
+const { get, uniq, isNull } = require('lodash')
+const falsey = require('falsey')
```

Comment:
do we really need this?

Reply:
i think better to set 0 - false, 1 - true

Reply:
I just reuse code from the serverSchema/Organization.js
As I see it come together (usage of pino & falsey)